float64[6] recommended_footholds
float64[2] height
float64    pitch_ref
float64[2] recommended_swing_times
float64[6] swing_foot_init_pos